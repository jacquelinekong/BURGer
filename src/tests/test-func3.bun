def null add(int a, int b){
	int r;
	r = a+b;
	println(r);

	r = a-b;
	println(r);

	r = a*b;
	println(r);

	if(a>>b){
		println("works");
	}
	if(a<<b){
		println("wrong");
	}
	if(a>=b){
		println("yes");
	}
	if(a<=b){
		println("no");
	}

	if(a!=b){
	 println("yas");
	}

}

add(46, 32);
