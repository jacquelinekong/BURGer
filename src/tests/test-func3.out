78
14
1472
works
yes
yas
